packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Comments/Comment.tsx

```diff
@@ -83,7 +84,18 @@ export const Comment = React.memo<{
               >
                 Mark as {comment.isResolved ? 'Unr' : 'r'}esolved
               </Menu.Item>
-              <Menu.Item onSelect={() => {}}>Share Comment</Menu.Item>
+              <Menu.Item
+                onSelect={() => {
+                  effects.browser.copyToClipboard(
+                    `${window.location.origin}${window.location.pathname}?comment=${comment.id}`
```

Comment:
I would put this behind a:

```ts
effects.router.createCommentUrl(comment.id)
```

Just move as much generic logic as possible behind more domain specific methods 😄

Reply:
Done :D